pages/affinity-diagramming.md

```diff
@@ -11,7 +11,7 @@ A way of finding themes in collections of ideas, quotes, or observations.
 
 ## Reasons to use it
 
-Quick and collaborative approach for drawing insights from qualitative data.
+To draw out insights from qualitative data quickly and collaboratively.
```

Comment:
I would have caught it in the "Copy edit" but maybe next time mention me/whoever is maintaining the print files so that it's easy to find the commit. Danke!